pkg/controller/elasticsearch/settings/merged_config_test.go

```diff
@@ -36,12 +37,17 @@ func TestNewMergedESConfig(t *testing.T) {
 		} `yaml:"network"`
 	}
 
+	policyCfg, _ := common.NewCanonicalConfigFrom(map[string]interface{}{
```

Comment:
There is `MustCanonicalConfig` which panics on error but which is good for testing.